HLTrigger/Configuration/python/HLT_75e33/tasks/ecalDigisTask_cfi.py

```diff
@@ -1,7 +1,7 @@
 import FWCore.ParameterSet.Config as cms
 
```

Comment:
Hi Marino, 
Sorry fighting with internet connection yesterday and too many discussions today... I am testing with the following command:
cmsDriver.py Phase2Removal -s HLT:75e33 --processName=HLTX --conditions auto:phase2_realistic_T21 --geometry Extended2026D88 --era Phase2C17I13M9 --eventcont
ent FEVTDEBUGHLT --filein=/store/relval/CMSSW_13_0_0_pre4/RelValTTbar_14TeV/GEN-SIM-RECO/130X_mcRun4_realistic_v2_2026D88noPU-v1/00000/2985fefc-4dd0-4c24-9950-f010ffe16ff3.r
oot -n 1000 --nThreads 1 --no_exec --inputCommands "keep *, drop *_hlt*_*_HLT, drop triggerTriggerFilterObjectWithRefs_l1t*_*_HLT" and it is working for me in CMSSW_13_0_0_pre4... 
I am not able to run the above proposed command due to the missing input file (Failed to open the file 'step2_SIM.root')
Any ideas?

Reply:
Hi Stephanie, no problem. One easy way to test a workflow with the Phase-2 HLT is
```
runTheMatrix.py -l 20834.76 # add -ne to only show the wf steps
```
On the other hand, the command you shared fails for me with this PR
```
  File "HLTrigger/Configuration/python/HLT_75e33/tasks/RawToDigiTask_cfi.py", line 9, in <module>
    from ..tasks.hltEcalDigisTask_cfi import *
ModuleNotFoundError: No module named 'HLTrigger.Configuration.HLT_75e33.tasks.hltEcalDigisTask_cfi'
```
which makes sense, because [this change](https://github.com/cms-sw/cmssw/pull/40912/files#diff-b7f6e00a71ed567138d25765eff704e7393ae734d30c4c9043bcb0ea63e4b55cR9) requires the file `hltEcalDigisTask_cfi.py`, and that file does not exist. Looks like you are either testing without the PR, or you have already fixed it locally but didn't commit and/or pushed the changes.

Reply:
Indeed I do not have hltEcalDigisTask_cfi in my directory ... I think I start to narrow it down but I need explainations: 
I did git cms-addpkg HLTrigger/Configuration/python/HLT_75e33/
But then I realised that I do not have locally HLTrigger.Configuration.HLT_75e33_cff.py... So probably it is taking it from the release and not from my local directory... Can it be the case that then it is using what is in the release and not my local directory so at the end I am not testing anything?
Thanks!

Reply:
> I did git cms-addpkg HLTrigger/Configuration/python/HLT_75e33/

Okay, I think this should be avoided. I don't know by heart what that does, but as a rule it's better to specify in cms-addpkg a whole package (as opposed to a subfolder of a package); in this case, this means

git cms-addpkg HLTrigger/Configuration